proto/Makefile

```diff
@@ -16,35 +16,22 @@
 # Author: Spencer Kimball (spencer.kimball@gmail.com)
 # Author: Andrew Bonventre (andybons@gmail.com)
 
-PROTO_LIB   := lib/libroachproto.a
-PROTOS      := api.proto config.proto data.proto errors.proto gossip.proto heartbeat.proto internal.proto
+PROTOC      := $(shell git rev-parse --show-cdup)_vendor/usr/bin/protoc
```

Comment:
Yes, we know where we are, but this felt a bit cleaner and now this line can be copied to the other proto Makefiles (e.g. wire.pb/Makefile). I'm fine with removing if you disagree.